main.go

```diff
@@ -71,6 +74,8 @@ var (
 	CacheMode = false
```

Comment:
In another pr this should be updated to match the ones above using the 0 value declaration rather than this declaration to false. The 0 value for bools is false but moreover like code should be consistent. Which means the new variable below should be defined as type string since the 0 value for string is "".

Reply:
Will also need to be unexported.

Reply:
I created this issue so we don't lose track of this technical debt.
See if you agree please.

https://github.com/rancher/ecm-distro-tools/issues/438